test/tests/mortar/continuity-2d-non-conforming/soln-continuity-pg.i

```diff
@@ -0,0 +1,120 @@
+[Mesh]
+  second_order = false
+  [file]
+    type = FileMeshGenerator
+    file = nodal_normals_test_offset_nonmatching_gap.e
+  []
+  [primary]
+    input = file
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = '2'
+    new_block_id = '20'
+  []
+  [secondary]
+    input = primary
+    type = LowerDBlockFromSidesetGenerator
+    sidesets = '1'
+    new_block_id = '10'
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
+[]
```

Comment:
Is this necessary? This may have been a copy-paste?

Reply:
Nice catch! Removed for both pg and non-pg tests here